packages/dev/core/src/Meshes/Builders/decalBuilder.ts

```diff
@@ -381,13 +381,8 @@ export function CreateDecal(
             let faceVertices: Nullable<DecalVertex[]> = oneFaceVertices;
 
             faceVertices[0] = extractDecalVector3(index, transformMatrix);
-            if (useLocalComputation) {
-                faceVertices[1] = extractDecalVector3(index + 2, transformMatrix);
-                faceVertices[2] = extractDecalVector3(index + 1, transformMatrix);
-            } else {
-                faceVertices[1] = extractDecalVector3(index + 1, transformMatrix);
-                faceVertices[2] = extractDecalVector3(index + 2, transformMatrix);
-            }
+            faceVertices[1] = extractDecalVector3(index + 1, transformMatrix);
+            faceVertices[2] = extractDecalVector3(index + 2, transformMatrix);

```

Comment:
I wonder why the existing code does not work anymore after the PR on the side orientation...

Reply:
Haha I was wondering WHY we needed that in the first hand. why do we need to change winding order if using local data?

Reply:
No idea, but it worked :)